src/Client.php

```diff
@@ -352,7 +353,9 @@ public function send($method, $payload, $path, $params = [], array $headers = []
         $response = call_user_func($this->middleware, $request, $response, $this);
 
         if ($response->getStatusCode() === 404) {
-            throw new Http\Exception\NotFoundException();
+            $content = json_decode($response->getBody()->getContents(), true);
+
+            throw new Http\Exception\NotFoundException($content['message'] ?? '');
```

Comment:
Can't we remove FQCN? I see that it is imported

Reply:
I've accidentally imported it. Removed import.